platform.md

```diff
@@ -579,6 +579,7 @@ Usage:
       - `CNB_LAYERS_DIR=<layers>`
       - `CNB_APP_DIR=<app>`
       - `PATH=/cnb/process:$PATH` where `$PATH` is the value of `$PATH` on the run-image.
+    - MUST have the working directory set to the value of `$CNB_APP_DIR`.
```

Comment:
```suggestion
    - MUST have the working directory set to `<app>`.
```
The working dir and `$CNB_APP_DIR` will match on the exported app image. But if the `-app` flag is provided it takes precedence over the build-time value of `$CNB_APP_DIR` and therefore `$CNB_APP_DIR` at build time may not match `$CNB_APP_DIR` in the exported image. Using `<app>` removes any ambiguity about whether we are referring to the build or launch time value.

Reply:
Resolved in https://github.com/buildpacks/spec/pull/187/commits/75ee979ea57b29487b6b676c71b0cc10fc5dfb2a